keyboards/neodox/info.json

```diff
@@ -0,0 +1,103 @@
+// Copyright 2022 MatteoDM <matteodalmo@gmail.com>
+// SPDX-License-Identifier: GPL-2.0-or-later
```

Comment:
this file doesn't need a license header, and comments like this cause some issues, currently, IIRC. 
```suggestion
```